app/components/case-studies-components/SimilarProjectSmallCard.tsx

```diff
@@ -5,39 +5,41 @@ import Image from "next/image";
 import { motion } from "framer-motion";
 
 import { truncateString } from "@/utils";
-import { FeaturedProjectCardProps } from "@/utils/interfaces";
+import { MainCaseStudyProps } from "@/utils/interfaces";
 
 import Button from "../Button";
 
-const SimilarProjectSmallCard = ({ project }: FeaturedProjectCardProps) => {
+const SimilarProjectSmallCard = ({ caseStudy }: MainCaseStudyProps) => {
   return (
     <div
-      key={project.title}
+      key={caseStudy.title}
       className="flex flex-col rounded-2xl bg-white900 dark:bg-black200 md:rounded-3xl"
     >
-      {project.figmaBannerImage && (
+      {caseStudy.figmaBannerImage && (
         <div className="flex md:h-[16rem]">
           <Image
-            src={project.figmaBannerImage}
+            src={caseStudy.figmaBannerImage.image}
             alt="Image of similar case study"
             className="h-full w-auto rounded-t-2xl md:rounded-t-3xl"
             style={{
               objectFit: "cover",
             }}
+            width={200}
+            height={256}
```

Comment:
Refer to my earlier comment about using width and height with "objectFit"... 
You're also misusing objectFit with Nextjs. Per the docs for Nextjs, The parent element must assign `position: "relative", position: "fixed", or position: "absolute"` style. It also states that for this to render correctly, you should apply `overflow-hidden` to the parent. 

You should be able to just use: `object-fit: "cover"` and not `objectFit` on line 25

Reply:
@cdsalty  I'm having the same issue here as mentioned earlier. I keep getting that error message from next when I don't have a height of width tag even with the object fit